### PR TITLE
[FEATURE][CU-86drct9ww] Differentiate/Identify users who came in via /pilot and via /c-org

### PR DESCRIPTION
**Pull request by cgarcia-lightit**
feat: add flow attribute to profiles and show it to profiles dashboard and into every profile as badge

## Tickets
- Click-Up: [86drct9ww](https://app.clickup.com/t/86drct9ww)

https://github.com/eocare/eo/assets/117183993/95de82b4-67d1-49fb-bff5-d910f3255fb8

Screenshot 2024-01-31 at 15 24 08

## Checks
- [x] Changes are backwards compatible with current iOS app
- [ ] API docs were regenerated if applicable
- [x] Requires coordination to deploy to production

### CODE REVIEW

**Comment by martinopp** [comment]
Task linked: [CU-86drct9ww Differentiate/Identify users who came in via /pilot and via /c-org](https://app.clickup.com/t/86drct9ww)

File: apps/eo_web/src/api/useApi.ts [hunk 1]
```diff
@@ -129,7 +129,6 @@ export const useApi = () => {
     await apiElixir.post<LaravelSuccessBase<unknown> | LaravelErrorValidation>(
       "/v2/profiles/eligible",
       { email },
-      authHeader,
```
Comment (sgoycoechea-lightit) [inline apps/eo_web/src/api/useApi.ts:132]: Is this intentional?

**Comment by sgoycoechea-lightit** [comment]
What happens if the user comes from the Employer Demo Center? Are we handling this appropiately?

**Comment by cgarcia-lightit** [comment]
> What happens if the user comes from the Employer Demo Center? Are we handling this appropiately?

In that case we should add a new flow. like `employer-demo`, but now we don't have it.

**Comment by sgoycoechea-lightit** [comment]
> > What happens if the user comes from the Employer Demo Center? Are we handling this appropiately?
> 
> In that case we should add a new flow. like `employer-demo`, but now we don't have it.

Shouldn't we add this before deploying? Or else it would be marked with a marketing site origin, right?

**Pull request merged**